charts/bulk-scan-payment-processor/values.yaml

```diff
@@ -23,7 +23,6 @@ java:
         - site-id-divorce
         - site-id-finrem
         - site-id-bulkscan
-        - payments-queue-listen-connection-string
```

Comment:
could you please confirm if you have removed secrets from key vault if it not used anymore ?

Reply:
secrets can be removed from vaults once they are not used in actual services. on that matter. i probably should deploy shared infra to all environments first and then current services (which is only this one ready)

but yep - from fault we can remove once not used. and that is shared infra again ;)